server/routes/bookings/bookingDetailsController.test.ts

```diff
@@ -2,148 +2,185 @@ import type { Express } from 'express'
 import request from 'supertest'
 import * as cheerio from 'cheerio'
 import { SessionData } from 'express-session'
+import { randomUUID } from 'crypto'
```

Comment:
Discovered that you can get a UUID (v4) directly from Node's `crypto` module so no need to have `uuid` module as an NPM dependency. Will sort this elsewhere in a subsequent PR...

Reply:
Hmm, could move the template repo over to use this also 🤔

Reply:
Yeah, I can have a look at that - can trial it on our projects to be sure it works!